runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -38,44 +38,61 @@ vim9script
 # The communication with gdb uses GDB/MI.  See:
 # https://sourceware.org/gdb/current/onlinedocs/gdb/GDB_002fMI.html
 
-# In case this gets sourced twice.
+def Echoerr(msg: string)
+  echohl ErrorMsg | echom $'[termdebug] {msg}' | echohl None
+enddef
+
+if !has('vim9script') ||  v:version < 900
+    # Needs Vim version 9.0 and above
+    Echoerr("You need at least Vim 9.0")
+    finish
+endif
+
+# Variables to keep their status among multiple instanced of Termdebug
+# Avoid to source the script twice.
 if exists('g:termdebug_loaded')
-  finish
+    Echoerr('Termdebug already loaded.')
```

Comment:
typicaly we don't error out, if a script is sourced twice. We just silently ignore this.

Reply:
True! I added it because when manually debugging one has to source the script many times and it often happens that the updated script is not sourced due to this guard and the developer may get mad to try to understand why a certain bug never go away. Adding this check considerably helped me. However, if you really want to remove it, no problems in removing it!  Just let me know :)